.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,56 @@
+### Instructions
+
+The github issue tracker is for bug reports, feature requests, and API documentation requests. General questions about how to use the GATK, how to interpret the output, etc. should be asked on the [official support forum](http://gatkforums.broadinstitute.org/gatk).
+- Search the existing github issues to see if your issue (or something similar) has already been reported.  If the issue already exists, you may comment there to inquire about the progress.
+- Determine whether your issue is a **bug report**, a **feature request**, or a **documentation request** (for tool gatkdocs only -- for forum docs see [the support issues repo](https://github.com/broadinstitute/dsde-docs/issues))
+- Consider if your "issue" is better addressed on the GATK forum: http://gatkforums.broadinstitute.org/gatk  Post there if you have questions about expected tool behavior, output format, unexpected results, or generally any question that does not fit into the categories above
+- Use a **concise** yet **descriptive** title
+- Choose the corresponding template block below and fill it in, replacing or deleting text in italics (surrounded by `_`) as appropriate
```

Comment:
If you're savvy enough to make feature requests for the engine, you probably can write a good issue without a template.  :-)  @droazen or @lbergelson thoughts?

Reply:
I see your point @ldgauthier, but I think @magicDGS's point is worth addressing.  The issue template is one of the first exposures people trying to contribute something have, so if it's an awkward match to what they're trying to tell us it is sort of dispiriting.  

What if we clarify it by changing the wording to `Affected Tools or Classes` that .  It's a bit more verbose but it makes it clear that it's either for top level bugs with tools or for API / internal bugs.

Reply:
@magicDGS Would that make it seem better for your use case?  Or do you have a different suggestion for an alternative template or something like that?

Reply:
I am ok with both the tool-driven and your proposal - I was just wondering what is the best way to submit an issue/proposal for the engine (as I do often), because I am planning to follow as much as possible the templates. I think that the addition of "Classes" might be good for letting know people that the template is the same for engine/tools.

Thanks for considering my opinion!